src/VirtualClient/VirtualClient.Main/profiles/PERF-SQL-POSTGRESQL.json

```diff
@@ -0,0 +1,54 @@
+{
+  "Description": "Postgresql Benchmark",
+  "Parameters": {
```

Comment:
Please take a look at one of the other profiles that exist in source at the moment. We are starting to add 'Metadata' to the profiles to make them more and more self-describing. As of now we are including 2 or 3 metadata properties that you can see in the other profiles. Please integrate similar ones into this profile when you have a sec. Thanks!